addons/website_slides_survey/models/slide_slide.py

```diff
@@ -62,6 +64,12 @@ def write(self, values):
             self._ensure_challenge_category(old_surveys=old_surveys - self.mapped('survey_id'))
         return result
 
+    def check_survey(self):
```

Comment:
Where is this function used?

Reply:
This function is no longer used, I am removing it.